### PR TITLE
Add tempest api job using single HCI node

### PR DESCRIPTION
**Pull request by viroel**
This zuul job inherits from ci-framework single node HCI job, which already enables manila by default and configures a ceph backend for it.
This new job should replace the current Prow job which don't deploy EDPM and Ceph.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1604

### CODE REVIEW

**Comment by viroel** [comment]
In tempest job, the openstack-operator and manila-operator images are being pulled from content provider:
`OPENSTACK_IMG": "38.102.83.27:5001/openstack-k8s-operators/openstack-operator-index:24d9eab6d0077a78163964a23e81e6e3b94f74f5"`
and also [manila operator](https://logserver.rdoproject.org/67/267/d060a1d3b92e01207214928ff2f934fd74df65a8/github-check/manila-operator-tempest/0e8ba8f/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack-operators/pods/manila-operator-controller-manager-5db89fd898-mltsj/manila-operator-controller-manager-5db89fd898-mltsj-describe):
  `  Image:         38.102.83.27:5001/openstack-k8s-operators/manila-operator:d060a1d3b92e01207214928ff2f934fd74df65a8`
  
  Tempest results:
  https://logserver.rdoproject.org/67/267/d060a1d3b92e01207214928ff2f934fd74df65a8/github-check/manila-operator-tempest/0e8ba8f/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

**Comment by viroel** [comment]
/hold
Adding a hold since we need the depends-on merged and tide will not be blocked by depends-on PRs open

**Comment by fmount** [comment]
awesome, thank you @viroel for starting this! I suspect I can close https://github.com/openstack-k8s-operators/manila-operator/pull/196 in favor of this patch, which is already green!

**Comment by fmount** [comment]
looks good to me, I'll merge once the dep lands

**Comment by viroel** [comment]
/hold cancel

the depends-on patch merged

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by fmount** [comment]
looks good [1]! @viroel we'll merge after beta freeze!

[1] https://logserver.rdoproject.org/67/267/e875de5ffa8390b963fc1cf747c96d3d9001d7ba/github-check/manila-operator-tempest/37aa4e2/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

**Comment by viroel** [comment]
recheck

lets see if is still working

**Comment by viroel** [comment]
hi @fmount, want to proceed with this now?

**Comment by fmount** [comment]
> hi @fmount, want to proceed with this now?

This is really great, I'll merge it but at least until GA let's keep both `prow` and `ci-framework`, and then we can decommission the old jobs!

**Review by fmount — APPROVED** [review]
/lgtm

**Pull request merged**
